lib/core/src/Cardano/Wallet/Api.hs

```diff
@@ -112,11 +113,21 @@ import Servant.API
 
 type Api t = CoreApi t :<|> CompatibilityApi t
 
-{-==============================================================================
-                                  Core API
-==============================================================================-}
+type CoreApi t =
+    Addresses t
+    :<|> Wallets
+    :<|> Transactions t
+    :<|> StakePools
+    :<|> Network
+
+type CompatibilityApi t =
+    DeleteByronWallet
+    :<|> GetByronWallet
+    :<|> GetByronWalletMigrationInfo
+    :<|> ListByronWallets
+    :<|> MigrateByronWallet
+    :<|> PostByronWallet
```

Comment:
Two points:

1. This change performs a general rearranging of API types (to which I have no objection!), but it's within a commit called "Implement API types for network information endpoint", which seems quite specific. Perhaps separate the rearrangement out into a separate commit, so that each commit does just one thing?

2. The "Core API" section comment is removed, but the "Compatibility API" section comment remains. Is that your intention?

Reply:
Yes. I just aligned things with the way the swagger document is also separated. Each section in Wallet Api reflect a group in swagger. Hence why I dropped "Core API", there's no such nesting in swagger. And I was actually tempted to change `Compatibility Api` with `Byron Wallets` to make it consistent with the swagger file too :grimacing:

Reply:
> replace Compatibility Api with Byron Wallets to make it consistent with the swagger file

That's actually a good alternative, I think. Though when I originally came up with that name my thinking was that the compatibility API might evolve over time to include more than just Byron wallets, and hence a more general name would be suitable.